cmd/posix.go

```
@@ -925,12 +925,14 @@ func deleteFile(basePath, deletePath string) error {
 
 	// Attempt to remove path.
 	if err := os.Remove(preparePath(deletePath)); err != nil {
+		if isSysErrNotEmpty(err) {
```

We need to add a comment.. on why this is an ignored error.

---

Commented added, ptal